scripts/validation/validate.js

```diff
@@ -18,16 +19,26 @@ const rootPath = path.join(__dirname, '../..');
 const MIN_DOC_LENGTH = 100;
 
 let serviceDeclarations;
+let modifiedServices;
 (async () => {
   try {
     serviceDeclarations = await loadServiceDeclarations(path.join(rootPath, config.get('serviceDeclarationsPath')));
+    modifiedServices = await getModifiedServices();
```

Comment:
Why do that even when it's not needed? Only compute modified services if it is requested! 🌱